### ISSUE TITLE
Broken links on website

### ISSUE THREAD

**Issue by janesmae** [PlasmaPy/plasmapy.github.io#16]
Hey, 

found some broken links on  the website:
-> About -> Acknowledging
-> Documentation -> Quick Start Guide
-> Team

All links referenced are in the main menu

**Comment by StanczakDominik** [comment]
Yeah, we really need to get around to improving the state of our website, it is pretty basic right now with most links being placeholders right now... We'll probably get something up there around the 0.1 release date. Thanks for the reminder!

**Comment by janesmae** [comment]
Agreed, but this should be a reminder and an actionable task at the same time. IMHO: Good for newcomers.

**Comment by parkerduckworth** [comment]
What would we like done with these broken links? Do these pages need to be created? Or do they exist and need to be routed correctly?

**Comment by ritiek** [comment]
Nope, these pages do not exist as of yet.

**Comment by StanczakDominik** [comment]
Relatively closed by #25 - #21 and #20 remain to be done but I'd rather not duplicate issues
